### PR TITLE
docs: Add CONTRIBUTING.md and PR Template

### LINKED ISSUE #3: docs: create CONTRIBUTING.md file
Create a `CONTRIBUTING.md` explaining the process to create an issue, and fork the repo. Also, you must include best practices for naming, since this is a docs repository. The name for issues can be docs or research. 

Any additional suggestions are welcome!

### PR DESCRIPTION
**Pull request by derianrddev**
# Pull Request Overview

📝 Summary
This PR adds two contributions to the project repository: a `CONTRIBUTING.md` file to establish clear contribution guidelines and a PR template to support the PR review process.

🔗 Related Issues
- Closes #3 

🔄 Changes Made
- Added `CONTRIBUTING.md`:
  - Provides a comprehensive guide for new and existing contributors to create issues, apply to issues, fork the repository, create branches, commit changes, and submit pull requests.
  - Establishes naming conventions for issues, branches, and commits using prefixes like `docs:` and `research:` to improve organization and clarity.

- Created a PR template:
  - Defines a structured format for PRs with sections such as:
    - **Summary**: Allows contributors to give a brief description of their PR.
    - **Related Issues**: Links any relevant issue.
    - **Changes Made**: A section to detail modifications and provide context for reviewers.
    - **Screenshots & Resources**: Where contributors can add visual aids or reference links.
    - **Comments**: For any additional information or questions.

🖼️ Screenshots & Resources
- **Preview of `CONTRIBUTING.md`**:
github com_derianrddev_semaphore-stellar-docs_blob_docs-add-contributing-file_CONTRIBUTING md

- **Preview of `.github/PULL_REQUEST_TEMPLATE.md`**:
image

💬 Comments
Thank you for the opportunity to contribute to this project.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+Make the necessary changes to the codebase.
+
+We encourage contributors to adhere to the following best practices:
+
+- Ensure that the content is clear and free of grammatical errors.
+- Maintain consistency in tone and formatting throughout the document.
+- Verify that all links work correctly and check for typos before submitting your changes.
+
+## Commit Your Changes
```
Comment (evgongora) [inline CONTRIBUTING.md:58]: Could you please add to use conventional commits and a link to the page :)

**Comment by evgongora** [comment]
Excellent job so far @derianrddev! Just some little comments :)

Could you please also add a testing section in the PR template, try to add if there is any testing/unit testing involved and any related screenshots or material and a simple self check list for contribution requirements

**Comment by derianrddev** [comment]
Here’s the result of the changes made:

Preview of the commit section in CONTRIBUTING.md:
image

Preview of the .github/PULL_REQUEST_TEMPLATE.md file:
image

**Comment by evgongora** [comment]
LGTM! Great job thank you :)

**Pull request merged**